core_tic_tac_toe/input_validator.py

```diff
@@ -19,6 +19,16 @@ def is_game_option_valid(self, game_option):
     def __check_range_of_options(self, game_option):
         return game_option >= 1 and game_option <= 4
 
+    def is_board_size_valid(self, board_size):
```

Comment:
I like the use of the `is_` prefix here. I noticed that you don't use it in the name of other predicate methods (methods that return a boolean value).

Reply:
Thank you! I need to be more consistent with this. I'm noticing I switch back and forth between `is` and `check`